libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/ApolloClient.kt

```diff
@@ -157,39 +154,20 @@ private constructor(
             // canBeBatched(apolloRequest.canBeBatched)
             addHttpHeader(ExecutionOptions.CAN_BE_BATCHED, apolloRequest.canBeBatched.toString())
           }
-          if (apolloRequest.ignorePartialData != null) {
-            ignorePartialData(apolloRequest.ignorePartialData)
-          }
-          if (apolloRequest.useV3ExceptionHandling != null) {
-            useV3ExceptionHandling(apolloRequest.useV3ExceptionHandling)
-          }
```

Comment:
This should be kept right?

Reply:
(maybe a test is missing around that, I feel this should have broken something (my bad probably! I introduced useV3ExceptionHandling))

Reply:
Yikes, thanks for noticing